draft-patton-cfrg-vdaf.md

```diff
@@ -705,6 +705,147 @@ TODO
 * Output shares are secret shares of a vector of field elements, each
   corresponding to a counter for one of the candidate prefixes.
 
+## Dependencies
+
+[TODO Clarify which of these can be assumed and which need to be explicitly defined]
+
+### Descriptions of finite fields
+We assume a structure for describing DPF output groups, named `ValueType`.
+For verifiable heavy hitters, we need an IDPF that at each level has two field 
+elements as outputs, and so the value type would be pairs of field elements.
+The IDPF key generation and evaluation functions will allow one `ValueType` parameter
+per bit in the IDPF domain, which allows us to use different types at the output
+layer, required for extractability. See [Section 4.3](https://eprint.iacr.org/2021/017.pdf) in [BBCGGI21] for details, and see [here](https://github.com/google/distributed_point_functions/blob/master/dpf/distributed_point_function.proto#L25) for an example implementation of `ValueType`. We use `ValueType` to refer to a description of a given group, and `Value` to refer to an actual element of the group.
+
+### Incremental Distributed Point Functions (IDPFs)
+An IDPF is defined over a domain of size `2**d`. The client can specify an index `alpha` and values `beta`, one for each level `l` in `[d]`. The key generation generates two DPF keys that individually hide `alpha` and `beta`. When locally evaluated at any point `x` in `2**l` at level `l`, the IDPF returns shares of `beta[l]`, if `x` is the `l`-bit prefix of `alpha`, and shares of zero otherwise.
```

Comment:
?
```suggestion
An IDPF is defined over a domain of size `2**d`. The client can specify an index `alpha` and values `beta`, one for each level `l` in `[d]`. The key generation generates two IDPF keys that individually hide `alpha` and `beta`. When locally evaluated at any point `x` in `2**l` at level `l`, the IDPF returns shares of `beta[l]`, if `x` is the `l`-bit prefix of `alpha`, and shares of zero otherwise.
```